### ISSUE TITLE
Azathioprine-induced Acute Pancreatitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6412]
**Preferred term label:**
Azathioprine-induced Acute Pancreatitis

**Synonyms**
Azathioprine pancreatitis

**Definition (free text, please give PubMed ID)**
Pancreatitis induced by the drug azathioprine.
PMID:  26468141 

**Parent term (use hpo.jax.org/app)**
Pancreatitis

**Comment by pnrobinson** [comment]
I think this is better conceptualized as a disease. I would suggest it is good for Mondo to have ``Drug-induced pancreatitis`` as a disease (here is a source for definitions https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4365846/). @nicolevasilevsky
